.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: python
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
+matrix:
+  include:
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8
+      env: TOXENV=py38
+    - env: TOXENV=lint
```

Comment:
I remain concerned about which Python version Travis will use when tox is configured for 3.6, but if it works, then that's great too.

Reply:
It works. I set the base python to py36 for lint.